infra/ansible/roles/build_srcs/tasks/main.yaml

```diff
@@ -17,6 +17,24 @@
     name: "{{ pytorch_wheels.files | map(attribute='path') }}"
     state: "forcereinstall"
 
+- name: Build torchvision
```

Comment:
Do you need to build torchvision from source? Or can you use the existing nightly CPU build of torchvision?

e.g. https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/250015f2394ae38e4f3ec4fa150ba33e72f237a7/tests/pytorch/nightly/common.libsonnet#L103-L104

Reply:
Right, building torchvision from source may not be a good idea. Removed the change.